op-service/client/backoff.go

```diff
@@ -0,0 +1,82 @@
+package client
+
+import (
+	"context"
+	"time"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/rpc"
+
+	"github.com/ethereum-optimism/optimism/op-service/backoff"
+)
+
+var (
+	// ExponentialBackoff is the default backoff strategy.
+	ExponentialBackoff = backoff.Exponential()
+)
+
+// InnerRPC is a minimal [client.RPC] interface that is used by the backoff client.
+//
+//go:generate mockery --name InnerRPC --output ./mocks/
+type InnerRPC interface {
```

Comment:
This is the same interface (& serves the same purpose) as https://github.com/ethereum-optimism/optimism/blob/d5ea28af16d92b2a18442d71736b5fba23757033/op-node/client/rpc.go#L21. Fine for now, but it should be removed when we do a larger cleanup of the sources.